fast-list/src/main/java/com/list/rados/fast_list/BaseList.kt

```diff
@@ -52,15 +52,24 @@ fun <T> RecyclerView.update(newItems: List<T>) {
 open class FastListAdapter<T>(private var items: MutableList<T>, private var list: RecyclerView
 ) : RecyclerView.Adapter<FastListViewHolder<T>>() {
 
-    private inner class BindMap(val layout: Int, var type: Int = 0, val bind: View.(item: T) -> Unit, val predicate: (item: T) -> Boolean)
+    private inner class BindMap(val layout: Int?, var type: Int = 0, val bind: View.(item: T) -> Unit, val predicate: (item: T) -> Boolean) {
```

Comment:
Why did you change the layout to be nullable? If you keep it as an Int, you won't have to add the !! down the line.